### DISCUSSION TITLE
Add pull request support for wiki pages

### DISCUSSION THREAD

**Discussion by Kangaroux** [community/community#50163] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

For repositories that make use of wikis, it would be great to have the ability to open pull requests for wiki changes. This enables public projects to accept changes from users, and for private projects that use the wiki internally it could be used as a review process.

**Comment by jpbochi** [comment]
Related links:
- https://stackoverflow.com/questions/10642928/how-can-i-make-a-pull-request-for-a-wiki-page-on-github
- https://github.com/isaacs/github/issues/846 (from Dec 22, 2016)
- https://www.growingwiththeweb.com/2016/07/enabling-pull-requests-on-github-wikis.html

**Comment by robtaylor** [comment]
Hi, Would love to see this as a core feature rather than us having to work around it!

**Comment by Ogglas** [comment]
This would be really good!

**Comment by Ecordonnier** [comment]
This is quite an important feature! Currently, the lack of PRs on wikis make them unusable for teams of developers.

**Comment by Zentheon** [comment]
I'm very surprised to find out Wiki PRs isn't just a thing. We need this!

**Comment by icyerasor** [comment]
Should be a no-brainer to have this, no? 
Can't think of a reason to not have this - and should be pretty straight-forward to implement / activate 🤔 ?

**Comment by Ecordonnier** [comment]
See also https://github.com/orgs/community/discussions/9473
